app/helpers/application_helper.rb

```diff
@@ -259,11 +259,6 @@ def view_to_url(view, parent = nil)
         action = 'show'
         return url_for_only_path(:action => action, :id => params[:id]) + "?display=generic_objects&generic_object_id="
       end
-      # If we do not want to use redirect or any kind of click action
-      if %w[Job VmdbDatabaseSetting VmdbDatabaseConnection VmdbIndex].include?(view.db) &&
```

Comment:
What about the `Job` here? I don't think it's part of the tree.

Reply:
@skateman this check is redundant, if you look at second part `%w[ops].include?(params[:controller])` this condition will never be true we do not show `Job` under OPS controller. Jobs were moved out to MiqTask controller long time ago